elide-core/src/test/java/com/yahoo/elide/ElideTest.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2018, Oath Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide;
+
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+public class ElideTest {
+
+    @Test
+    public void testPathNormalization() {
+
+        String normalizedPath;
+
+        normalizedPath = Elide.parse("/books").getText();

```

Comment:
Add the exact same tests you have but invert "/" so it is "\".  I imagine these would break Elide.